packages/expo-gl/src/types.ts

```diff
@@ -0,0 +1,7 @@
+// copied from: https://github.com/microsoft/TypeScript/blob/bc76e7f03c6a80706246f7c35e392bfc88a93ac3/lib/lib.dom.d.ts#L17355-L17361
+export interface WebGLObject {}
```

Comment:
this was removed from `lib.dom.ts` in v4.2 - copied from the old `lib.dom.ts`

Reply:
Maybe add this comment in the source code as well? 🤔 I wonder what the reason for this removal is?